acrobot.py

```diff
@@ -10,18 +10,22 @@
 from pydrake.all import *
 from ilqr import IterativeLinearQuadraticRegulator
 import time
+import utils_derivs_interpolation
 
 ####################################
 # Parameters
 ####################################
 
-T = 1.5        # total simulation time (S)
-dt = 1e-2      # simulation timestep
+T = 10       # total simulation time (S)
+dt = 0.004      # simulation timestep
+
+meshcat = StartMeshcat()
 
 # Solver method
 # must be "ilqr" or "sqp"
 method = "ilqr"
-MPC = True      # MPC only works with ilqr for now
+MPC = False                         # MPC only works with ilqr for now
+MESHCAT_VISUALISATION = True        # Visualisation with meshcat or drake visualizer
```

Comment:
nit, I'd prefer lower case for `meshcat_visualization` (MPC is only capitalized b/c it's an acronym). 

Also, maybe you already knew this, but meldis allows you to view DrakeVisualizer stuff in the browser, similar to meshcat.

Reply:
Hi Vince, I am actually not familiar with meldis, I think i might have tried it at the start when i was first using drake but couldnt seem to get it working? Is it better than the meshcat visualisation I do, should i try learn how to use it again?